src/user-federation/messages.json

```
@@ -92,6 +92,11 @@
     "userFedDisableConfirmTitle": "Disable user federation provider?",
     "userFedDisableConfirm": "If you disable this user federation provider, you cannot make changes to the provider.",
 
+    "removeImportedUsers": "Remove imported users?",
+    "removeImportedUsersMessage": "Do you really want to remove all imported users? The option \"Unlink users\" makes sense just for the Edit Mode \"Unsynced\" and there should be a warning that \"unlinked\" users without the password in the Keycloak database won't be able to authenticate.",
```

This dialog message seems a bit too verbose. I think the second part can be removed with the understanding that once "Unlink Users" is implemented, the dialog can specify the password authentication warning there? Those are  my thoughts ... WDYT? I know this is how it looks in the mocks but I think we could potentially clean it up somehow
```suggestion
    "removeImportedUsersMessage": "Do you really want to remove all imported users? 
```

---

I completely agree that the wording is extremely weak, but I think it's a slippery slope on editing the text from the mocks because I can see making tens if not hundreds of changes. I think we leave that to the doc resource and/or a text sweep later in the project.

---

Sounds like a good plan!